app/decorators.py

```diff
@@ -0,0 +1,37 @@
+from collections.abc import Mapping
+from functools import wraps
+
+from flask import abort, request
+
+
+def json_payload_view(acceptable_content_types=("application/json",)):
```

Comment:
We use `"text/plain"` as well when we're using this - do we need it on the other one (if so should it be a default as well)?

Reply:
The only reason the other one has `text/plain` is because SNS is a little bit :loop: y and sends its requests as `text/plain` - there's nothing we can do. The rest of our APIs manage to enforce `application/json` as we're in charge of the requests there. So I'm just following suit.